Common/Data/Consolidators/RenkoConsolidator.cs

```diff
@@ -119,62 +147,39 @@ public RenkoConsolidator(decimal barSize, Func<IBaseData, decimal> selector, Fun
         /// <summary>
         /// Gets the bar size used by this consolidator
         /// </summary>
-        public decimal BarSize
-        {
-            get { return _barSize; }
-        }
+        public decimal BarSize => _barSize;
 
         /// <summary>
         /// Gets the most recently consolidated piece of data. This will be null if this consolidator
         /// has not produced any data yet.
         /// </summary>
-        public IBaseData Consolidated
-        {
-            get; private set;
-        }
+        public IBaseData Consolidated { get; private set; }
 
         /// <summary>
         /// Gets a clone of the data being currently consolidated
         /// </summary>
-        public IBaseData WorkingData
-        {
-            get { return _currentBar == null ? null : _currentBar.Clone(); }
-        }
+        public IBaseData WorkingData => _currentBar?.Clone();
 
         /// <summary>
         /// Gets the type consumed by this consolidator
         /// </summary>
-        public Type InputType
-        {
-            get { return typeof (IBaseData); }
-        }
+        public Type InputType => typeof(IBaseData);
 
         /// <summary>
         /// Gets <see cref="RenkoBar"/> which is the type emitted in the <see cref="IDataConsolidator.DataConsolidated"/> event.
         /// </summary>
-        public Type OutputType
-        {
-            get { return typeof(RenkoBar); }
-        }
+        public Type OutputType => typeof(RenkoBar);
 
         // Used for unit tests
-        internal RenkoBar OpenRenkoBar
-        {
-            get
-            {
-                return new RenkoBar(null, _openOn, _closeOn,
-                    BarSize, _openRate, _highRate, _lowRate, _closeRate);
-            }
-        }
+        internal RenkoBar OpenRenkoBar => new RenkoBar(null, _openOn, _closeOn, _barSize, _openRate, _highRate, _lowRate, _closeRate);
 
         private void Rising(IBaseData data)
         {
             decimal limit;
 
-            while (_closeRate > (limit = (_openRate + BarSize)))
+            while (_closeRate > (limit = _openRate + _barSize))
```

Comment:
>NO CHANGE REQUIRED

Interesting choice to convert these usages, also somewhat surprised we didn't turn them into automatic, get-only property, since `_barSize` is read-only already.

Reply:
I left `BarSize` there, because it is a public property and removing it could break user's code.

Reply:
of course! I wasn't recommending removing the `BarSize` property, was just stating that it appears we don't even need a `_barSize` field, since we could define the property as:
``` csharp
public decimal BarSize { get; }
```

and then have everyone reference that property and completely remove the `_barSize` field -- but no matter, what's here is great!